dashboard/test/models/pd/application/facilitator1920_application_test.rb

```diff
@@ -105,10 +108,18 @@ class Facilitator1920ApplicationTest < ActiveSupport::TestCase
       application = build :pd_facilitator1920_application, form_data_hash: application_hash, course: :csf
 
       answers = application.full_answers
-      assert answers.key? :csf_availability
-      assert answers.key? :csf_partial_attendance_reason
-      refute answers.key? :csd_csp_fit_availability
-      refute answers.key? :csd_csp_teachercon_availability
+      [
+        :csf_good_standing_requirement,
+        :csf_summit_requirement,
+        :csf_workshop_requirement,
+        :csf_community_requirement
+      ].each {|x| assert answers.key? x}
+      [
+        :csd_csp_lead_summer_workshop_requirement,
```

Comment:
add `csd_csp_good_standing_requirement` and double check against final version of required fields